README.md

```diff
@@ -22,6 +22,33 @@ A tool for signing and verifying npm packages.
 * Support offline and online signature verification
 * Retrieve root cert and public keys via TUF client
 
+## Prerequisites
+
+- Node.js version 16+ (LTS)
+- Add `node_modules/.bin` to your `$PATH` to access the installed `tsc` bin
```

Comment:
@bdehamer added these basic setup requirements while adding the docker prereq. Is this what you ended up doing as well with `PATH`?

Reply:
I've never done this. I think the convention here is to just use `npx` (`npm exec`) to invoke project-local binaries.